### PR TITLE
coreboot-utils: 4.14 -> 4.19

### PR DESCRIPTION
**Pull request by jmbaur**
###### Description of changes

Utils changes:
- https://review.coreboot.org/plugins/gitiles/coreboot/+/4d661eebe95b06acae7d99777ed36ba56a560112/Documentation/releases/coreboot-4.19-relnotes.md#additional-coreboot-changes
- https://review.coreboot.org/plugins/gitiles/coreboot/+/4d661eebe95b06acae7d99777ed36ba56a560112/Documentation/releases/coreboot-4.17-relnotes.md#util_cbmem_add-flamegraph_compatible-timestamps-output
- https://review.coreboot.org/plugins/gitiles/coreboot/+/4d661eebe95b06acae7d99777ed36ba56a560112/Documentation/releases/coreboot-4.17-relnotes.md#util_cbmem_add-an-option-to-append-timestamp

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/coreboot-utils/default.nix [hunk 1]
```diff
@@ -84,6 +84,8 @@ let
     amdfwtool = generic {
       pname = "amdfwtool";
       meta.description = "Create AMD firmware combination";
+      buildInputs = [ openssl ];
+      nativeBuildInputs = [ pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/coreboot-utils/default.nix:88]:
```suggestion
      nativeBuildInputs = [ pkg-config ];
```
that was an alias unless you mean the python package

File: pkgs/tools/misc/coreboot-utils/default.nix [hunk 2]
```diff
@@ -105,13 +107,16 @@ let
...
         runHook postInstall
       '';
-      postFixup = let
-        binPath = [ coreutils acpica-tools gnugrep gnused file ];
-      in "wrapProgram $out/bin/acpidump-all --set PATH ${lib.makeBinPath binPath}";
+      postFixup =
+        let
+          binPath = [ coreutils acpica-tools gnugrep gnused file ];
+        in
+        "wrapProgram $out/bin/acpidump-all --set PATH ${lib.makeBinPath binPath}";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/coreboot-utils/default.nix:114]:
```suggestion
      postFixup = ''
        wrapProgram $out/bin/acpidump-all \
          --set PATH ${lib.makeBinPath [ coreutils acpica-tools gnugrep gnused file ]}
      '';
```

File: pkgs/tools/misc/coreboot-utils/default.nix (hunk 1 above)
Comment (jmbaur) [reply pkgs/tools/misc/coreboot-utils/default.nix]: Thanks!

File: pkgs/tools/misc/coreboot-utils/default.nix (hunk 2 above)
Comment (jmbaur) [reply pkgs/tools/misc/coreboot-utils/default.nix]: Applied this change

**Pull request merged**